src/default.js

```diff
@@ -29,6 +29,28 @@ import VueRouter   from 'vue-router';
 Vue.use(VueEvents);
 Vue.use(VueRouter);
 
+// --- Gettext ----
+
+import GetTextPlugin from 'vue-gettext';
+import translations from '../l10n/translations.json'
+
+Vue.use(GetTextPlugin, {
+  availableLanguages: {
+<<<<<<< HEAD
```

Comment:
merge conflict :see_no_evil:

Reply:
i tried to squash all those commits into the last 3 but this seems like it didn't work out well...

Reply:
worst case: start a new branch ;-)

Reply:
okay will do next time

Reply:
please resolve it in here or recreate. let me know if you need support.
I honestly don't want to merge this in in the current state ;-)